File: python/bubble_sort.py
def bubbleSort(numbers):
    for i in range(0, len(numbers) - 1):
        for j in range(0, len(numbers) - 1):
            if (numbers[j] > numbers[j + 1]):
                temp = numbers[j]
                numbers[j] = numbers[j + 1]
                numbers[j + 1] = temp
                
def main():
    numbers = [10, 20, 5, 15, 10]
    bubbleSort(numbers)
    print(numbers)

if __name__ == "__main__":
    main()

File: js/data-structures/Array/bubbleSort.js
/**
 * Sorting Algorithm: Bubble Sort
 * Time Complexity: O(n^2)
 */
let bubbleSort = function (array) {
  for (let i = 0; i < array.length - 1; i++) {
    for (let j = 0; j < array.length - 1; j++) {
      if (array[j] > array[j + 1]) {
        let temp = array[j];
        array[j] = array[j + 1];
        array[j + 1] = temp;
      }
    }
  }
};

let array = [1, 6, 4, 5, 9, 3, 10, 16, 12, 11];
bubbleSort(array);
console.log(array);
